grpc/oob/machine/machine.go

```diff
@@ -254,6 +254,12 @@ func (m Action) PowerSet(ctx context.Context, action string) (result string, err
 	client := bmclib.NewClient(host, "623", user, password, bmclib.WithLogger(m.Log))
 
 	err = client.Open(ctx)
+	if err != nil {
```

Comment:
Looks like this duplicates some lines ([L267-L275](https://github.com/tinkerbell/pbnj/pull/111/files#diff-58e9fa0e549a798eb5175bffdc3b82999c98e43ac5ada9340d5b2f93babb117aR267-R275)) below. Would you be open to maybe using those lines here instead? We'd probably still want to get the span and status message.

Reply:
sure

Reply:
done

Reply:
thanks! looks like we are still duplicating the `err != nil` check block from ([L267-L275](https://github.com/tinkerbell/pbnj/pull/111/files#diff-58e9fa0e549a798eb5175bffdc3b82999c98e43ac5ada9340d5b2f93babb117aR267-R275)). Maybe remove those lines? Other options? What do you think?

Reply:
Hey @nhorman, following up, I think with this error handling now in the more correct place we would want to remove the unneeded lines below: [270-278](https://github.com/tinkerbell/pbnj/blob/ddb83c5fa7f665d623e992f7ca69ad42e137c685/grpc/oob/machine/machine.go#L270-L278).